com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -258,22 +263,37 @@ internal void EndSceneEvent(uint sceneEventId)
         /// <summary>
         /// Gets the scene name from full path to the scene
         /// </summary>
+        /// <returns>scene name</returns>
         internal string GetSceneNameFromPath(string scenePath)
         {
             var begin = scenePath.LastIndexOf("/", StringComparison.Ordinal) + 1;
             var end = scenePath.LastIndexOf(".", StringComparison.Ordinal);
             return scenePath.Substring(begin, end - begin);
         }
 
+        /// <summary>
+        /// Returns the full path of the scene and scene name as it
+        /// is displayed in the Build Settings Scenes in Build list.
+        /// </summary>
+        /// <param name="scenePath"></param>
+        /// <returns>full path and scene name</returns>
+        internal string GetScenePathAndName(string scenePath)
+        {
+            return scenePath.Replace("Assets/", "").Replace(".unity", "");
```

Comment:
Is this always true or could it be it happens to be true for the projects we set up?  Then I wonder if Unity has an API for this.  Like I see [this](https://docs.unity3d.com/ScriptReference/SceneManagement.Scene-path.html).  Is this where we don't have an actual scene to query, just paths?  I just want to challenge to make sure there isn't a more standard way of doing this

Reply:
I believe the Assets/ folder is always true for the scope of V1.0.0, when we get to addresables this all will have to change...but at that point we hopefully will be working with a SceneManagerAPI derived NetworkSceneManager so much of this code will no longer exist.

The scenePath is coming from the SceneUtility.GetScenePathByBuildIndex which will return the full path to the scene asset that will include the Assets/ and the <scene name>.unity extension.

Reply:
what would happen if a user had a scene named like this: `Assets/assets.unity/ArtAssets/someScene.unity`

`GetScenePathAndName` would rename that to `Assets/assets/Art/SomeScene.unity`

Reply:
This is handled and there is an additional test to verify that.  👍